package.json

```diff
@@ -77,6 +77,7 @@
     "lightning": "^4.10.6",
     "ln-service": "^52.8.0",
     "lodash": "^4.17.21",
+    "lodash.merge": "^4.6.2",
```

Comment:
why add an additional package if we are not removing the main lodash package?

Reply:
because we need to switch also other lodash subpackage. for now this PR removed the `{merge} from lodash` but there are other to be removed in later PR.